js/app-super-resolution/instructions.md

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+
+// Licensed under the MIT license.
```

Comment:
the copyright banner should only be applied to source code files

Reply:
👍 thanks